### PR TITLE
Adding VPX Express custom script

### PR DESCRIPTION
**Pull request by vivekkumac**

### CODE REVIEW

File: experimental/vpx_express/custom_script.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+import base64
+import requests
+import sys
+
+
+# Globals
+AZURE_CUST_DATA_FILE = "/nsconfig/.AZURE/customdata"
+
+# Set logging
+logging.basicConfig()
```
Comment (akshayhavile) [inline experimental/vpx_express/custom_script.py:18]: Is all log redirected to console? Can we append it to single separate file?

File: experimental/vpx_express/custom_script.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        time.sleep(30)
+
+    logger.info("read command line parameters")
+    if len(sys.argv) < 2:
+        logger.error("password is required for nitro calls")
+        exit(1)
+
+    try:
+        logger.info("intialize variables")
+        usr = cdata['username']
```
Comment (akshayhavile) [inline experimental/vpx_express/custom_script.py:226]: If we have plan to include ssh key as part of authentication mechanism, then we need to consider different username and password. (internal user created.)

**Review by akshayhavile — COMMENTED** [review]
Few comments added regarding enhancement and log file

File: experimental/vpx_express/custom_script.py (hunk 1 above)
Comment (vivekkumac) [reply experimental/vpx_express/custom_script.py:18]: logs are being finally written to /var/log/azure/../extension.log

**Pull request closed without merging**